cns/cniconflist/testdata/fixtures/azure-windows-swift-overlay.conflist

```diff
@@ -0,0 +1,52 @@
+{
+	"cniVersion": "0.3.0",
```

Comment:
Here is an example overlay conflist today in AKS, from a cluster that I just made once:
```
{
    "cniVersion":  "0.3.0",
    "name":  "azure",
    "adapterName":  "",
    "plugins":  [
                    {
                        "type":  "azure-vnet",
                        "mode":  "bridge",
                        "bridge":  "azure0",
                        "capabilities":  {
                                             "portMappings":  true,
                                             "dns":  true
                                         },
                        "ipam":  {
                                     "type":  "azure-cns",
                                     "mode":  "overlay"
                                 },
                        "dns":  {
                                    "Nameservers":  [
                                                        "10.0.0.10",
                                                        "168.63.129.16"
                                                    ],
                                    "Search":  [
                                                   "svc.cluster.local"
                                               ]
                                },
                        "AdditionalArgs":  [
                                               {
                                                   "Name":  "EndpointPolicy",
                                                   "Value":  {
                                                                 "Type":  "OutBoundNAT",
                                                                 "ExceptionList":  [
                                                                                       "10.244.0.0/16"
                                                                                   ]
                                                             }
                                               },
                                               {
                                                   "Name":  "EndpointPolicy",
                                                   "Value":  {
                                                                 "Type":  "OutBoundNAT",
                                                                 "ExceptionList":  [
                                                                                       "fddc:be24:d690:87d::/64"
                                                                                   ]
                                                             }
                                               },
                                               {
                                                   "Name":  "EndpointPolicy",
                                                   "Value":  {
                                                                 "Type":  "ACL",
                                                                 "Protocols":  "6",
                                                                 "Action":  "Block",
                                                                 "Direction":  "Out",
                                                                 "RemoteAddresses":  "168.63.129.16/32",
                                                                 "RemotePorts":  "80",
                                                                 "Priority":  200,
                                                                 "RuleType":  "Switch"
                                                             }
                                               },
                                               {
                                                   "Name":  "EndpointPolicy",
                                                   "Value":  {
                                                                 "Type":  "ACL",
                                                                 "Action":  "Allow",
                                                                 "Direction":  "In",
                                                                 "Priority":  65500
                                                             }
                                               },
                                               {
                                                   "Name":  "EndpointPolicy",
                                                   "Value":  {
                                                                 "Type":  "ACL",
                                                                 "Action":  "Allow",
                                                                 "Direction":  "Out",
                                                                 "Priority":  65500
                                                             }
                                               }
                                           ],
                        "windowsSettings":  {
                                                "enableLoopbackDSR":  true
                                            }
                    }
                ]
}
```